enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -375,7 +375,11 @@ def can_redeem(self, learner_id, content_key):
         Checks if the given learner_id has a number of existing subsidy transactions
         LTE to the learner enrollment cap declared by this policy.
         """
-        if len(self.transactions_for_learner(learner_id)['transactions']) < self.per_learner_enrollment_limit:
+        if self.per_learner_enrollment_limit is None:
+            return super().can_redeem(learner_id, content_key)
+
+        learner_transactions_count = len(self.transactions_for_learner(learner_id)['transactions'])
+        if learner_transactions_count < self.per_learner_enrollment_limit:
```

Comment:
Previously, the default for `self.per_learner_enrollment_limit` was `0`, which prevents the policy from being redeemed even though this policy intends to have no per-learner limits.